cloudmanager/resource_netapp_cloudmanager_connector_gcp.go

```diff
@@ -140,7 +150,16 @@ func resourceOCCMGCPCreate(d *schema.ResourceData, meta interface{}) error {
 	occmDetails.SubnetID = d.Get("subnet_id").(string)
 	occmDetails.MachineType = d.Get("machine_type").(string)
 	occmDetails.ServiceAccountEmail = d.Get("service_account_email").(string)
-	client.GCPServiceAccountPath = d.Get("service_account_path").(string)
+	service_account_path := d.Get("service_account_path").(string)
+	if service_account_path == "" {
+		service_account_key, err := ioutil.ReadFile(service_account_path)
+		if err != nil {
+			return fmt.Errorf("Cannot read service account file: %s", err)
+		}
+		client.GCPServiceAccountKey = string(service_account_key)
+	} else {
```

Comment:
I would check that service_account_key is not empty.   And report an error if both service_account_key and service_account_path are not specified.